tools/fuzzy_diff.sh

```diff
@@ -0,0 +1,7 @@
+MYTMPDIR="$(mktemp -d)"
```

Comment:
any readme about what is this and how to use it?

Reply:
Hi, the tool is for visualizing the differences in fuzzy sections like in weblate, the output is as follows:
<img width="812" alt="image" src="https://github.com/user-attachments/assets/91a7aa25-4e5e-4a0c-a1f4-d3d2e68caa1c" />

Reply:
Please also make note in README.md instructing how to use it. Why does it work only on the first fuzzy string?

Reply:
All right, but at least mention the script's existence in readme.